graphql/util.go

```diff
@@ -59,40 +60,48 @@ func getGroupedFiles(ctx context.Context, name string, taskID string, execution
 	return &GroupedFiles{TaskName: &name, Files: apiFileList}, nil
 }
 
-func setScheduled(ctx context.Context, sc data.Connector, taskID string, isActive bool) (*restModel.APITask, error) {
+func setScheduled(ctx context.Context, sc data.Connector, isActive bool, taskIDs ...string) ([]*restModel.APITask, error) {
 	usr := mustHaveUser(ctx)
-	t, err := task.FindOneId(taskID)
+	tasks, err := task.FindAll(db.Query(task.ByIds(taskIDs)))
 	if err != nil {
 		return nil, ResourceNotFound.Send(ctx, err.Error())
 	}
-	if t == nil {
-		return nil, ResourceNotFound.Send(ctx, errors.Errorf("task %s not found", taskID).Error())
+	if tasks == nil {
+		return nil, ResourceNotFound.Send(ctx, errors.Errorf("task IDs %s not found", strings.Join(taskIDs, ",")).Error())
 	}
-	if t.Requester == evergreen.MergeTestRequester && isActive {
-		return nil, InputValidationError.Send(ctx, "commit queue tasks cannot be manually scheduled")
+	taskPtrs := []*task.Task{}
+	for _, t := range tasks {
+		taskPtrs = append(taskPtrs, &t)
+		if t.Requester == evergreen.MergeTestRequester && isActive {
+			return nil, InputValidationError.Send(ctx, "commit queue tasks cannot be manually scheduled")
```

Comment:
Do we want to use a catcher here so that the behavior mimics what the behavior was when we had a for loop in the resolver? Or is returning  when one errors and not attempting to schedule the rest the preferred setup?

Reply:
I would expect all of the tasks in the same version to have the same requester?